src/platform/i386/timer.c

```diff
@@ -54,7 +54,7 @@
 #define TN_FSB_EN_CNF	(1ll << 14)	/* 1 = deliver interrupts via FSB instead of APIC */
 #define TN_FSB_INT_DEL_CAP	(1ll << 15)	/* read only, 1 = FSB delivery available */
 
-#define HPET_INT_ENABLE 0x3	/* Sets ENABLE_CNF and LEG_RT_CNF */
+#define HPET_INT_ENABLE(n) (0x1 << n)	/* Clears the INT n for level-triggered mode. */
```

Comment:
I'm a little confused why these changes are in this patch?  Did you get them from Jakob?